packages/client/bin/startRpc.ts

```diff
@@ -33,24 +33,28 @@ type RPCArgs = {
 }
 
 /**
- * Returns a jwt secret from a provided file path, otherwise saves a randomly generated one to datadir
+ * Returns a jwt secret from a provided file path, otherwise saves a randomly generated one to datadir if none already exists
  */
 function parseJwtSecret(config: Config, jwtFilePath?: string): Buffer {
-  let jwtSecret
-  if (jwtFilePath !== undefined) {
-    const jwtSecretContents = readFileSync(jwtFilePath, 'utf-8').trim()
+  let jwtSecret: Buffer
+  const defaultJwtPath = `${config.datadir}/jwtsecret`
+  if (jwtFilePath !== undefined || existsSync(defaultJwtPath)) {
+    const jwtSecretContents = readFileSync(jwtFilePath ?? defaultJwtPath, 'utf-8').trim()
```

Comment:
Check if `jwtFilePath` exists? I don't know what it will throw otherwise, but probably a less-descriptive error (user might not know that this jwt does not exist)

Reply:
Good point, I think that if jwtFilePath is provided but does not exist, we should throw (and not default to the other one, as that would likely confuse users).

Reply:
Resolved!

Reply:
Just checked, if file does not exist, you get an error: 

```
fs.readFileSync('abacdsadf.ts')
Uncaught Error: ENOENT: no such file or directory, open 'abacdsadf.ts'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'abacdsadf.ts'
```

(But for the user, it might not be directly clear that this is their `jwt` stuff, so it would be more descriptive to just throw `jwt: file does not exists`, or something like that)

Reply:
This should not happen with the latest update though right? Since we're only reading if either the first exists, or the second exists.